core/src/main/java/com/linecorp/armeria/common/ImmutableHttpHeaders.java

```diff
@@ -89,6 +89,16 @@ public HttpHeaders status(HttpStatus status) {
         return unsupported();
     }
 
+    @Override
+    public MediaType contentType() {
+        return delegate.contentType();
+    }
+
+    @Override
+    public HttpHeaders contentType(MediaType mediaType) {
+        return delegate.contentType(mediaType);
```

Comment:
this should be unsupported()?

Reply:
Oh I see, sorry my mistake. Its supposed to be Immutable